validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -86,8 +98,8 @@ def __init__(self,
                  thread_pool=None):
         """Initialize the BlockValidator
         Args:
-            block_cache: The cache of all recent blocks and the processing
-                state associated with them.
+            block_manager: The componenet that stores blocks and maintains
```

Comment:
typo s/componenet/component/

Reply:
Nice catch. I will fix this on my next push of updates.